src/fix-sami-041.js

```diff
@@ -0,0 +1,112 @@
+// Author(s): Nicholas Volk
+
+//import createDebugLogger from 'debug';
+import clone from 'clone';
+
+import {fieldToString} from './utils.js';
+
+// const debug = createDebugLogger('@natlibfi/marc-record-validators-melinda:fix-sami-041');
+
+/* 'sma': eteläsaame, 'sme': pohjoissaame, 'smj': luulajansaame, 'smn': inarinsaame, 'sms': koltansaame */
+const samiLanguages = ['sma', 'sme', 'smj', 'smn', 'sms'];
+const relevantSubfieldCodes = ['a', 'd']; // Subfield codes that should also have 'smi' if a sami langauge is used. Confirmed by A.R. via Slack 2025-12-05
```

Comment:
Theis can be inside of default function

Reply:
and is compiler-wise more ineffective that way (not that it really matters).